standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -342,15 +342,17 @@ public CompactionInfo findNextToCompact(FindNextCompactRequest rqst) throws Meta
   public void markCompacted(CompactionInfo info) throws MetaException {
     try {
       Connection dbConn = null;
-      Statement stmt = null;
+      PreparedStatement pstmt = null;
```

Comment:
use try-with-resources since you refactored this method, it's reported in findbugs